pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
     "numpy~=1.21",
     "scipy",
     "ase",
-    "node-graph==0.2.19",
+    "node-graph @ git+https://github.com/superstar54/node-graph.git@use_wrapt_proxy",
```

Comment:
This is only used to test. will change it back when node-graph publishes a new version.